codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/RulesBasedAuthSchemeResolverGenerator.kt

```diff
@@ -93,9 +94,7 @@ class RulesBasedAuthSchemeResolverGenerator {
                         // SigV4A case
                         write("case .sigV4A(let param):")
                         indent()
-                        // sigv4a trait is not yet implemented by Smithy
-                        // This is a SDK-level customization until the trait is added
-                        write("var sigV4Option = AuthOption(schemeID: \"${SigV4Trait.ID}a\")")
+                        write("var sigV4Option = AuthOption(schemeID: \"${SigV4ATrait.ID}\")")
```

Comment:
nit: write using Smithy formatters.
```
write("var sigV4Option = AuthOption(schemeID: \$S)", SigV4ATrait.ID)
```
(`$S` is the Smithy formatter for a string literal, it adds the surrounding double quotes for you)

Reply:
Changed as suggested.